portfolio_website.html

```diff
@@ -17,44 +17,191 @@
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="flex project btn-flex">
+                    <li class="proj-btn">html</li>
+                    <li class="proj-btn">bootstrap</li>
+                    <li class="pproj-btn">Ruby</li>
+                 </ul>
+                </div>
+                <a href="/" class="cursor project-link">See Project</a>
+             </li>
+             <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+             </li>
+             <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+            </li>
+            <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="proj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+           </li>
+            </ul>
+          </section>
+          <section class="about-section">
```

Comment:
I suggest you use flexbox for aligning contents in this section as stated in the project requirement